runtime-modules/content/src/benchmarks/benchmarking.rs

```diff
@@ -518,6 +509,167 @@ benchmarks! {
+    // - channel has all feature paused except the necessary for the extr. to succeed to maximize permission validation complexity
+    withdraw_from_channel_balance {
+        let (channel_id, group_id, lead_account_id, _, _) =
+            setup_worst_case_scenario_curator_channel::<T>(0,T::StorageBucketsPerBagValueConstraint::get().min as u32,T::DistributionBucketsPerBagValueConstraint::get().min as u32, false)?;
+        let origin= RawOrigin::Signed(lead_account_id.clone());
+        Pallet::<T>::set_channel_paused_features_as_moderator(
+            origin.clone().into(),
+            crate::ContentActor::Lead,
+            channel_id,
+            super::all_channel_pausable_features_except(BTreeSet::from_iter(vec![crate::PausableChannelFeature::ChannelFundsTransfer])),
+            b"reason".to_vec(),
+        ).unwrap();
+
+        let amount = <T as balances::Config>::Balance::from(100u32);
+        let _ = Balances::<T>::deposit_creating(
+            &ContentTreasury::<T>::account_for_channel(channel_id),
+            amount + T::ExistentialDeposit::get(),
+        );
+
+        let actor = crate::ContentActor::Lead;
+        let origin = RawOrigin::Signed(lead_account_id.clone());
+    }: _ (origin, actor, channel_id, amount)
+        verify {
+            assert_eq!(
+                T::CouncilBudgetManager::get_budget(),
+                amount,
+            );
+
+            assert_eq!(
+                Balances::<T>::usable_balance(ContentTreasury::<T>::account_for_channel(channel_id)),
+                T::ExistentialDeposit::get(),
+            );
+        }
+
+    // Worst case scenario:
+    // - curator channel belonging to a group with max number curator and max curator permissions
+    // - channel has all feature paused except the necessary for the extr. to succeed to maximize permission validation complexity
+    claim_channel_reward {
+        let h in 1 .. MAX_MERKLE_PROOF_HASHES;
+
+        let cumulative_reward_claimed: BalanceOf<T> = 100u32.into();
+        let payments = create_pull_payments_with_reward::<T>(2u32.pow(h), cumulative_reward_claimed);
+        let commitment = generate_merkle_root_helper::<T, _>(&payments).pop().unwrap();
+        let proof = build_merkle_path_helper::<T, _>(&payments, 0);
+        let (channel_id, group_id, lead_account_id, _, _) =
+            setup_worst_case_scenario_curator_channel::<T>(0,T::StorageBucketsPerBagValueConstraint::get().min as u32,T::DistributionBucketsPerBagValueConstraint::get().min as u32, false)?;
+        let origin = RawOrigin::Signed(lead_account_id.clone());
+
+        Pallet::<T>::set_channel_paused_features_as_moderator(
+            origin.clone().into(),
+            crate::ContentActor::Lead,
+            channel_id,
+            super::all_channel_pausable_features_except(BTreeSet::from_iter(vec![
+                crate::PausableChannelFeature::CreatorCashout,
+            ])),
+            b"reason".to_vec(),
+        ).unwrap();
```

Comment:
This is a nice feature, I think it's worth factoring out to another utility function like `set_channel_paused_features_all_except(features)` so that it can be used in other benchmarks too (without too much boilerplate)